tests/scale_test/notifier.py

```diff
@@ -171,6 +179,11 @@ def once(ctx, uuid, version):
         ctx.obj['workflow_name'],
         ctx.obj['es_query_path'],
     )
+    if ctx.obj['force']:
+        if label is None:
+            label = {'force': None}
+        else:
+            label['force'] = None
```

Comment:
Why `None`?

Reply:
My thought was that providing a value could lead to the interpretation that the value had a meaning. We only care that the Cromwell label force exists, not that it has any value.